core/templates/self_list.h

```diff
@@ -41,6 +41,9 @@ class SelfList {
 		SelfList<T> *_first = nullptr;
 		SelfList<T> *_last = nullptr;
 
+		// Forbid copying, which has broken behavior.
+		void operator=(const List &p_other) {}
```

Comment:
What do you think about keeping if `public` but `deleted`? More idiomatic, perhaps.

Reply:
I was intending to but matched the class template, I'm okay with either way and I prefer the `delete` syntax myself, more informative

Will update